docs/typescript/how-to-create-a-temporal-client-in-typescript.md

```diff
@@ -0,0 +1,140 @@
+---
+id: how-to-create-a-temporal-client-in-typescript
+title: How to create a Temporal Client in Typescript
+sidebar_label: Create a temporal Client
+description: Create a temporal Client
+tags:
+  - developer-guide
+  - sdk
+  - typescript
+---
+
+Use a new `WorflowClient()` with the requisite gRPC `Connection` to create a new Client.
+
+```typescript
+import { Connection, WorkflowClient } from '@temporalio/client';
+const connection = new Connection(); // to configure for production
+const client = new WorkflowClient(connection.service);
+```
+
+Declaring the `WorflowClient()` creates a new connection to the Temporal service.
+
+If you ommit the connection and just call the `new WorkflowClient()`, you will create a default connection that works locally. However, configure your connection and Namespace when [deploying to production](typescript/security#encryption-in-transit-with-mtls).
+
+The following example, creates a Client, connects to an account, and declares your Namespace.
+
+```typescript
+import { Connection, WorkflowClient } from '@temporalio/client';
+
+const connection = new Connection({
+  address: '<Namespace ID>.tmprl.cloud', // defaults port to 7233 if not specified
+  tls: {
+    // set to true if TLS without mTLS
+    // See docs for other TLS options
+    clientCertPair: {
+      crt: clientCert,
+      key: clientKey,
+    },
+  },
+});
+await connection.untilReady();
+const client = new WorkflowClient(connection.service, {
+  namespace: 'your.namespace',
+});
+```
+
+A full example for Workers looks like this:
+
+```typescript
+import { Worker, NativeConnection } from '@temporalio/worker';
+import * as activities from './activities';
+
+async function run() {
+  const connection = await NativeConnection.create({
+    address: 'foo.bar.tmprl.cloud', // defaults port to 7233 if not specified
+    tls: {
+      // set to true if TLS without mTLS
+      // See docs for other TLS options
+      clientCertPair: {
+        crt: clientCert,
+        key: clientKey,
+      },
+    },
+  });
+
+  const worker = await Worker.create({
+    connection,
+    namespace: 'foo.bar', // as explained in Namespaces section
+    // ...
+  });
+  await worker.run();
+}
+
+run().catch((err) => {
+  console.error(err);
+  process.exit(1);
+});
+```
+
+[The Hello World mTLS sample](https://github.com/temporalio/samples-node/tree/main/hello-world-mtls/) demonstrates sample code used to connect to a Temporal Cloud account.
```

Comment:
As a h4 of Create Temporal Clients section - we need to have a Client Options / Call headers section that shows how to provide the options needed to connect to a custom env or Cloud.